src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -13,36 +13,44 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
+
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Login</title>
-  <link rel="stylesheet" href="/css/main.css">
+  <title>CodeU Chat App</title>
+  <link rel="stylesheet" href="/css/loginpage.css">
 </head>
 <body>
 
-  <%@ include file="../component/navbar.jsp" %>
+  <nav>
```

Comment:
This don't necessarily be needed to be a nav tag. Could just be a div or header since there is really no navigation here. Also consider adding a background image to make it look more fun or sleek depending on the look and feel you prefer :)